tests/prague/eip7692_eof_v1/eip3540_eof_v1/code_validation_function.py

```diff
@@ -1,189 +0,0 @@
-"""
-Code validation of CALLF, RETF opcodes tests
-"""
-
-from typing import List
-
-from ethereum_test_tools.eof.v1 import Container, Section
-from ethereum_test_tools.eof.v1.constants import MAX_CODE_SECTIONS
-from ethereum_test_tools.vm.opcode import Opcodes as Op
-
-VALID: List[Container] = [
-    Container(
-        name="retf_code_input_output",
-        sections=[
-            Section.Code(
-                code=Op.PUSH0 + Op.CALLF[1] + Op.POP + Op.POP + Op.STOP,
-                code_inputs=0,
-                code_outputs=0,
-                max_stack_height=2,
-            ),
-            Section.Code(
-                code=Op.PUSH0 + Op.RETF,
-                code_inputs=1,
-                code_outputs=2,
-                max_stack_height=2,
-            ),
-        ],
-    ),
-    Container(
-        name="stack_height_equal_code_outputs_retf_zero_stop",
```

Comment:
this test is being removed, but looks legit, can you confirm removal is intended?